projects/kernel-config/Makefile

```diff
@@ -12,7 +12,7 @@
 HASH?=$(shell git ls-tree HEAD -- ../$(notdir $(CURDIR)) | awk '{print $$3}')
 # Name on Hub
 IMAGE:=kernel
-ARCH?=$(shell if [ "$$(arch)" = "x86_64" ]]; then echo x86; else $$(arch); fi)
+ARCH?=x86_64
```

Comment:
The idea is that it's as easy as defining `kernel_config.${ARCH}` and include all the arch-specific config there. The non-arch config lives in `kernel_config.base`, so all of that will be the same across all LinuxKit kernels.